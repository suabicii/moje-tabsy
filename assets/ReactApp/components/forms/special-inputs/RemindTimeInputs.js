import React from "react";
import RemindTimeInput from "./RemindTimeInput";

function RemindTimeInputs({amount, handleUnitInputChange}) {
    const inputArray = [];

    for (let i = 0; i < amount; i++) {
        inputArray.push(<RemindTimeInput handleUnitInputChange={handleUnitInputChange} ordinalNumber={i + 1} />);
    }

    return (
        <>
            {inputArray.map((item, index) => (
                <div key={index}>
                    {item}
                </div>
            ))}
        </>
    );
}

export default RemindTimeInputs;