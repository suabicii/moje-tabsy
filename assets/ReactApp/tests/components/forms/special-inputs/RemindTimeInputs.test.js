import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import RemindTimeInputs from "../../../../components/forms/special-inputs/RemindTimeInputs";

it('should correctly render remind time input', () => {
    const renderer = new ReactShallowRenderer();
    const handleUnitInputChange = jest.fn();
    renderer.render(<RemindTimeInputs handleUnitInputChange={handleUnitInputChange} amount={1}/>);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});