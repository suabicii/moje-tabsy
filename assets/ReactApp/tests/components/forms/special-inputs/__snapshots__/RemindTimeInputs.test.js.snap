// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render remind time input 1`] = `
<React.Fragment>
  <div>
    <RemindTimeInput
      handleUnitInputChange={[MockFunction]}
      ordinalNumber={1}
    />
  </div>
</React.Fragment>
`;
